system-addon/content-src/styles/_theme.scss

```diff
@@ -0,0 +1,80 @@
+  --newtab-background-color: $grey-10;
+  --newtab-text-primary-color: $grey-90;
+  --newtab-text-secondary-color: $grey-50;
+  --newtab-text-conditional-color: $grey-60;
+  --newtab-link-primary-color: $blue-60;
+  --newtab-link-secondary-color: $teal-70;
+  --newtab-border-primary-color: $grey-40;
+  --newtab-border-secondary-color: $grey-30;
+  --newtab-icon-primary-color: $grey-90-80;
+  --newtab-icon-secondary-color: $grey-90-60;
+  --newtab-icon-tertiary-color: $grey-30;
+  --newtab-textbox-border: $grey-90-20;
+  --newtab-textbox-color: inherit;
+  --newtab-active-element-color: $grey-20-60;
+  --newtab-button-primary-color: $blue-60;
+  --newtab-button-secondary-color: inherit;
+  --newtab-contextmenu-button-color: $white;
+  --newtab-modal-color: $white;
+  --newtab-overlay-color: $grey-20;
+  --newtab-searchbox-background-color: $white;
+  --newtab-searchbox-border-color: transparent;
+  --newtab-searchbox-text-color: inherit;
+  --newtab-searchbox-icon-color: $grey-90-40;
+  --newtab-section-header-text-color: $grey-50;
+  --newtab-section-navigation-text-color: $grey-50;
+  --newtab-section-navigation-link-color: $blue-40;
+  --newtab-section-active-contextmenu-color: $grey-90;
+  --newtab-topsites-label-color: inherit;
+  --newtab-card-background-color: $white;
+  --newtab-inner-box-shadow-color: $black-10;
+}
+
+// Dark theme
+.outer-wrapper.dark-theme {
+  --newtab-background-color: $grey-80;
+  --newtab-text-primary-color: $grey-10;
+  --newtab-text-secondary-color: $grey-10-40;
+  --newtab-text-conditional-color: $grey-10;
+  --newtab-link-primary-color: $blue-40;
+  --newtab-link-secondary-color: $blue-40;
+  --newtab-border-primary-color: $grey-10-80;
+  --newtab-border-secondary-color: $grey-10-40;
+  --newtab-icon-primary-color: $grey-10-80;
+  --newtab-icon-secondary-color: $grey-10-40;
+  --newtab-icon-tertiary-color: $grey-10-20;
+  --newtab-textbox-border: $grey-10-40;
+  --newtab-textbox-color: $grey-80;
+  --newtab-active-element-color: $grey-60;
+  --newtab-button-primary-color: $blue-60;
+  --newtab-button-secondary-color: $grey-70;
+  --newtab-contextmenu-button-color: $grey-80;
+  --newtab-modal-color: $grey-90;
+  --newtab-overlay-color: $grey-80;
+  --newtab-searchbox-background-color: $grey-60;
+  --newtab-searchbox-border-color: $grey-10-40;
+  --newtab-searchbox-text-color: $grey-10-60;
+  --newtab-searchbox-icon-color: $grey-10-60;
+  --newtab-section-header-text-color: $grey-10-80;
+  --newtab-section-navigation-text-color: $grey-10-80;
+  --newtab-section-active-contextmenu-color: $white;
+  --newtab-topsites-label-color: $grey-10-80;
+  --newtab-card-background-color: $grey-90;
+  --newtab-inner-box-shadow-color: $grey-60;
+}
+
+// scss variables related to the theme.
+$border-primary: 1px solid var(--newtab-border-primary-color);
+$border-secondary: 1px solid var(--newtab-border-secondary-color);
+$input-primary: $blue-60;
+$input-secondary: transparent;
+$shadow-primary: 0 0 0 5px $grey-30;
```

Comment:
I would have expected any shade of black to be different for light vs dark. Default background is $grey-10, so this $grey-30 shadow is 20 degrees away. To maintain the same amount of contrast on the dark theme's $grey-80 background, I would have expected $grey-60 for the shadow:

60 ![screen shot 2018-03-23 at 4 40 01 pm](https://user-images.githubusercontent.com/438537/37857584-61a0278c-2eb9-11e8-9e6a-fe593427d06d.png)

30 ![screen shot 2018-03-23 at 4 40 22 pm](https://user-images.githubusercontent.com/438537/37857589-7faf7336-2eb9-11e8-911c-c58d9ef8c4bd.png)

@bryanbell ?

Reply:
ah, Amy (is she on github?) said the box shadow on hover could stay the same.

<img width="478" alt="screen shot 2018-03-23 at 7 48 51 pm" src="https://user-images.githubusercontent.com/36629/37857655-505ba892-2ed3-11e8-8d58-90f94a7e237d.png">

Reply:
bryan said "Top is better by far" with +1 from aaron